dspace-spring-rest/src/main/java/org/dspace/app/rest/converter/ItemConverter.java

```diff
@@ -22,11 +28,34 @@
 public class ItemConverter extends DSpaceObjectConverter<org.dspace.content.Item, org.dspace.app.rest.model.ItemRest> {
 	@Autowired
 	private CollectionConverter collectionConverter;
+	@Autowired
+	private BitstreamConverter bitstreamConverter;
 
 	@Override
 	public ItemRest fromModel(org.dspace.content.Item obj) {
 		ItemRest item = super.fromModel(obj);
-		// item.setOwningCollection(collectionConverter.fromModel(obj.getOwningCollection()));
+		item.setInArchive(obj.isArchived());
+		item.setDiscoverable(obj.isDiscoverable());
+		item.setWithdrawn(obj.isWithdrawn());
+		item.setLastModified(obj.getLastModified());
+		try {
+			//item.setTemplateItemOf(collectionConverter.fromModel(obj.getTemplateItemOf()));
+		} catch (Exception e) {
+			e.printStackTrace();
+		}
```

Comment:
I realize this is a WIP. But, we should either clean up empty try/catch statements before merging, OR add a TODO comment as to why these are currently commented out (if they are dependent on some future code). There's two of these in this file.